drivers/ovsSwitch.go

```diff
@@ -95,7 +95,7 @@ func NewOvsSwitch(bridgeName, netType, localIP string, fwdMode string,
 			datapath = "vrouter"
 		default:
 			log.Errorf("Invalid datapath mode")
-			return nil, errors.New("Invalid forwarding mode. Expects 'bridge' or 'routing'")
+			return nil, errors.New("invalid forwarding mode. Expects 'bridge' or 'routing'")
```

Comment:
The point of this change is that error messages are meant to start with a lowercase letter and not have any punctuation at the end.

The goal of this convention is to be able to compose errors nicely. If you run into an error, you can do something like `return fmt.Errorf("encountered error while opening file: %v", err); this error would be composable even further, e.g.: "failed to save data: encountered error while opening file: file doesn't exist".

There are some language conventions which should be followed. Others follow these conventions as well. I didn't come up with these conventions.